task/github-request-reviewers/0.1/README.md

```diff
@@ -0,0 +1,70 @@
+# Request reviewers on a pull request
+
+The `github-request-reviewers` task let one request reviewers on a pull request. It implements the corresponding [GitHub API](https://docs.github.com/en/rest/reference/pulls#request-reviewers-for-a-pull-request) to request reviewers.
+
+## Changelog
+
+- v0.1: Initial version.
+
+## Install the Task
+
+```
+kubectl apply -f https://raw.githubusercontent.com/tektoncd/catalog/main/task/github-request-reviewer/0.1/github-request-reviewers.yaml
+```
+
+## Secrets
+
+This Task requires access to a GitHub token. The token is expected to be in a workspace, which will typically be bound to Kubernetes Secret at run time. The name of the file in the workspace is `token` by default, but it can be configured via the `github-token-key` parameter from the [parameters](#parameters) described below.
+
+To create such a Secret via `kubectl`:
+
+```
+kubectl create secret generic github --from-literal token="MY_TOKEN"
+```
+
+Check [this](https://help.github.com/en/github/authenticating-to-github/creating-a-personal-access-token-for-the-command-line) to get personal access token for `Github`.
+
+See GitHub's documentation on [Understanding scopes for OAuth Apps](https://developer.github.com/apps/building-oauth-apps/understanding-scopes-for-oauth-apps/) to figure out what scopes you need to give to this token to add comment to an issue or a pull request.
```

Comment:
```suggestion
See GitHub's documentation on [Understanding scopes for OAuth Apps](https://developer.github.com/apps/building-oauth-apps/understanding-scopes-for-oauth-apps/) to figure out what scopes you need to give to this token to add a comment to an issue or a pull request.
```

Reply:
Thanks - in fact this comment had to be updated from add a comment to request a reviewer 🙏